src/Evolution/DiscontinuousGalerkin/Limiters/Weno.hpp

```diff
@@ -115,6 +115,15 @@ class Weno<VolumeDim, tmpl::list<Tags...>> {
     static constexpr OptionString help = {
         "Linear weight for each neighbor element's solution"};
   };
+  /// \brief The TVB constant for the minmod TCI
+  ///
+  /// See `Limiters::Minmod` documentation for details.
+  struct TvbConstant {
```

Comment:
This feature is very useful! thanks for adding it.

Reply:
Awesome, glad to hear it! 🙂